pkg/accelerator/accelDevice_test.go

```diff
@@ -26,6 +27,16 @@ import (
 	. "github.com/onsi/gomega"
 )
 
+var pciAddr = "0000:00:00.1"
```

Comment:
not a fan of globals in this case.

can you move pciAddr to be within Describe statement and define newPciDevice inline within Describe as well to be used by the tests.

Reply:
e.g

```
Describe("some thing", func() {
    pciAddr := "00000:00:00.1"
    newPciDeviceFn := func() *ghw.PciDevice { return &ghw.PciDevice{....} }

}
```